packages/grafana-runtime/src/utils/queryResponse.test.ts

```diff
@@ -277,6 +278,57 @@ describe('Query Response parser', () => {
     expect(ids).toEqual(['A', 'B']);
   });
 
+  describe('Cache notice', () => {
+    let resp: any;
+
+    beforeEach(() => {
+      resp = {
+        url: '',
+        type: 'basic',
+        config: { url: '' },
+        status: 200,
+        statusText: 'OK',
+        ok: true,
+        redirected: false,
+        headers: new Headers(),
+        data: {
+          results: {
+            A: { frames: [{ schema: { fields: [] } }] },
+          },
+        },
+      };
+    });
+
+    test('adds notice for responses with X-Cache: HIT header', () => {
+      const queries: DataQuery[] = [{ refId: 'A' }];
+      resp.headers.set('X-Cache', 'HIT');
+      expect(toDataQueryResponse(resp, queries).data[0].meta.notices).toStrictEqual([
+        { severity: 'info', text: 'Cached response' },
```

Comment:
**Suggestion:** maybe the text `"Cached response"` could be a const defined in the `queryResponse.ts`, so its value is only defined in a single place.

Reply:
added a const for the full notice in [bfd007e](https://github.com/grafana/grafana/pull/45564/commits/bfd007e51fcd7929ffc116b8425c7ab25e48cd10)